tdrs-backend/docker-compose.yml

```diff
@@ -76,21 +89,27 @@ services:
     image: grafana/grafana:11.2.0
     ports:
       - 9400:9400
+    environment:
+      - GF_PATHS_PROVISIONING=/usr/share/grafana/conf/provisioning
+      - GF_SERVER_HTTP_PORT=9400
     volumes:
-      - ./plg/grafana/datasources.yml:/etc/grafana/provisioning/datasources/default.yml
-      - ./plg/grafana/dashboards/provider.yml:/etc/grafana/provisioning/dashboards/default.yml
-      - ./plg/grafana/dashboards:/var/lib/grafana/provisioning/dashboards
-      - ./plg/grafana/custom.ini:/etc/grafana/grafana.ini
+      - ./plg/grafana/datasources.local.yml:/usr/share/grafana/conf/provisioning/datasources/datasources.yml
+      - ./plg/grafana/providers.local.yml:/usr/share/grafana/conf/provisioning/dashboards/providers.yml
+      - ./plg/grafana/dashboards:/var/lib/grafana/dashboards
+      - ./plg/grafana/custom.local.ini:/usr/share/grafana/conf/custom.ini
```

Comment:
Could this be simplified to something like this?

`./plg/grafana/:/usr/share/grafana/`

Reply:
Not without running custom commands on the container to create the same structure. It is easier to use the directory structure the container already provides for us.